src/main/java/io/lettuce/core/AbstractRedisAsyncCommands.java

```diff
@@ -1571,6 +1571,16 @@ public void shutdown(boolean save) {
         return dispatch(commandBuilder.unlink(keys));
     }
 
+    @Override
+    public RedisFuture<Boolean> copy(K source, K destination) {
+        return dispatch(commandBuilder.copy(source, destination));
+    }
+
+    @Override
+    public RedisFuture<Boolean> copy(K source, K destination, int destinationDd) {
```

Comment:
It would make sense to introduce a `CopyArgs` object that includes `destinationDb` and `replace` options.

Reply:
I didn't notice COPY has replace argument :) CopyArgs added (builder, comments etc.)